python/brayns/cli/core/service_cli.py

```diff
@@ -35,14 +35,14 @@ class ServiceCli(Cli):
 
     def register(self, parser: argparse.ArgumentParser) -> None:
         parser.add_argument(
-            '--port',
```

Comment:
As a personal preference, I believe --port was clearer (and shorter), and the doc (if needed) already specifies that its the braynsService port.